<?xml version="1.0"?>
<launch>

  <group ns="quadrotor1" clear_params="true">
     <node name="dummy_sensor_multi" pkg="prototype_5" type="dummy_sensor_multi" />
     <node name="DKF_multi" pkg="prototype_5" type="DKF_multi" />
     <node name="DBA_bis" pkg="prototype_5" type="DBA_bis" />
  </group>

  <group ns="quadrotor2" clear_params="true">
     <node name="dummy_sensor_multi" pkg="prototype_5" type="dummy_sensor_multi" />
     <node name="DKF_multi" pkg="prototype_5" type="DKF_multi" />
     <node name="DBA_bis" pkg="prototype_5" type="DBA_bis" />
  </group>

  <group ns="quadrotor3" clear_params="true">
     <node name="dummy_sensor_multi" pkg="prototype_5" type="dummy_sensor_multi" />
     <node name="DKF_multi" pkg="prototype_5" type="DKF_multi" />
     <node name="DBA_bis" pkg="prototype_5" type="DBA_bis" />
  </group>

  <group ns="quadrotor4" clear_params="true">
     <node name="dummy_sensor_multi" pkg="prototype_5" type="dummy_sensor_multi" />
     <node name="DKF_multi" pkg="prototype_5" type="DKF_multi" />
     <node name="DBA_bis" pkg="prototype_5" type="DBA_bis" />
  </group>

  <group ns="quadrotor5" clear_params="true">
     <node name="dummy_sensor_multi" pkg="prototype_5" type="dummy_sensor_multi" />
     <node name="DKF_multi" pkg="prototype_5" type="DKF_multi" />
     <node name="DBA_bis" pkg="prototype_5" type="DBA_bis" />
  </group>

  <group ns="quadrotor6" clear_params="true">
     <node name="dummy_sensor_multi" pkg="prototype_5" type="dummy_sensor_multi" />
     <node name="DKF_multi" pkg="prototype_5" type="DKF_multi" />
     <node name="DBA_bis" pkg="prototype_5" type="DBA_bis" />
  </group>



  <group ns="communication" clear_params="true">
     <node name="communication_node_multi" pkg="prototype_5" type="communication_node_multi" />
     <node name="swarm_node_multi" pkg="prototype_5" type="swarm_node_multi" />
  </group>

  <group ns="evaluation" clear_params="true">
     <node name="evaluation_node_multi" pkg="prototype_5" type="evaluation_node_multi" />
  </group>




</launch>
